### DISCUSSION TITLE
Pascal-P6/doc/the_p6_compiler.docx  unreadable?

### DISCUSSION THREAD

**Discussion by tomcircuit** [samiam95124/Pascal-P6#101] [General]
I've tried desktop MS WORD (2019) and some browser version of MS WORD viewer, as well as an online .docx to PDF converter, and none are able to open this file.  Can anyone else open this file? Worst case, could it be PDF'd?

**Comment by samiam95124** [comment]
Agreed, I'll take care of this tonight (save it to PDF).

The workaround for this is to use Libreoffice, the same word processor I use.

The background on this is that when Microsoft moved to a subscription model for word, I would have dramatically increased my costs for word, since I used to only update it every 5 years or so. I moved to Libreoffice, which works fine by the way on these documents. Even though I save it in word format, figured it was only a matter of time before it became incompatible with Word, and indeed that happened.

**Comment by samiam95124** [comment]
Done and pushed to repo.

Moving this issue to discussions so others can read it.
